playpen/ansible/roles/dev/tasks/debug.yml

```diff
@@ -0,0 +1,89 @@
+# All tasks in this file operate conditionally based on the truthiness of the pulp_dev_debug var
+---
+# Tools handy for debugging that aren't generally useful in normal development
+- name: Install debugging tools
+  dnf: name={{ item }} state=present
+  with_items:
+    - gdb
+    - lsof
+    - tcpdump
+
+# The crazy list of packages below was not generated by hand.
+# To generate it, first get all the debuginfo packages that gdb demands installed
+# Then, "rpm -qa | grep debuginfo | sed 's/debuginfo\(.*\)/debuginfo/' | sort"
+# The sed strips the EVRA off of the package NEVRA so it's not such a pain to maintain
+- name: Install debuginfo packages for gdb
+  dnf:
+    name: "{{ item }}"
+    state: present
+    enablerepo: fedora-debuginfo, updates-debuginfo
+  with_items:
+    - acl-debuginfo
+    - attr-debuginfo
+    - audit-debuginfo
+    - cracklib-debuginfo
+    - curl-debuginfo
+    - cyrus-sasl-debuginfo
+    - deltarpm-debuginfo
+    - e2fsprogs-debuginfo
+    - elfutils-debuginfo
+    - expat-debuginfo
+    - file-debuginfo
+    - freetype-debuginfo
+    - gdbm-debuginfo
+    - glib2-debuginfo
+    - glibc-debuginfo
+    - glibc-debuginfo
+    - gpgme-debuginfo
+    - jbigkit-debuginfo
+    - keyutils-debuginfo
+    - krb5-debuginfo
+    - lcms2-debuginfo
+    - libassuan-debuginfo
+    - libcap-debuginfo
+    - libdb-debuginfo
+    - libffi-debuginfo
+    - libgcrypt-debuginfo
+    - libgpg-error-debuginfo
+    - libidn-debuginfo
+    - libjpeg-turbo-debuginfo
+    - libpng-debuginfo
+    - libselinux-debuginfo
+    - libssh2-debuginfo
+    - libtiff-debuginfo
+    - libverto-debuginfo
+    - libwebp-debuginfo
+    - libxml2-debuginfo
+    - libxslt-debuginfo
+    - lua-debuginfo
+    - m2crypto-debuginfo
+    - ncurses-debuginfo
+    - nspr-debuginfo
+    - nss-debuginfo
+    - nss-softokn-debuginfo
+    - nss-util-debuginfo
+    - openjpeg2-debuginfo
+    - openldap-debuginfo
+    - openssl-debuginfo
+    - pam-debuginfo
+    - pcre-debuginfo
+    - popt-debuginfo
+    - pygpgme-debuginfo
+    - pyliblzma-debuginfo
+    - python-billiard-debuginfo
+    - python-debuginfo
+    - python-ldap-debuginfo
+    - python-lxml-debuginfo
+    - python-pillow-debuginfo
+    - python-pycurl-debuginfo
+    - python-pymongo-debuginfo
+    - python-simplejson-debuginfo
+    - pyxattr-debuginfo
+    - readline-debuginfo
+    - rpm-debuginfo
+    - saslwrapper-debuginfo
+    - sqlite-debuginfo
+    - util-linux-debuginfo
+    - xz-debuginfo
+    - yum-metadata-parser-debuginfo
+    - zlib-debuginfo
```

Comment:
Yeah, when I saw this list was when I thought "we're going to need to put a conditional on this...